### PR TITLE
Add docs for business systems api

### PR DESCRIPTION
**Pull request by hebime**
Docs for our new favorite route 😜

### CODE REVIEW

**Comment by alavers** [comment]
http://starwars.wikia.com/wiki/Kashyyyk

File: slate/source/includes/rest/core/_app_user.md [hunk 1]
```diff
@@ -308,7 +310,52 @@ curl https https://api.smooch.io/v1/apps/5963c0d619a30a2e00de36b8/appusers/deb92
...
+```
+
+> Response:
+
+```
+200 OK
+```
+```json
+{
+    "bizSystems": [
```
Comment (alavers) [inline slate/source/includes/rest/core/_app_user.md]: oopsie

Comment (hebime) [reply slate/source/includes/rest/core/_app_user.md]: uhhhg copy 🍝

File: slate/source/includes/rest/core/_app_user.md [hunk 2]
```diff
@@ -308,7 +310,52 @@ curl https https://api.smooch.io/v1/apps/5963c0d619a30a2e00de36b8/appusers/deb92
...
+            "type": "slack",
+            "channelId": "C872AE91B"
+        },
+        {
+            "type": "zendesk",
+            "ticketId": "9999"
+        },
+        {
+            "type": "hipchat",
+            "roomId": 1337
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_app_user.md]: Are these actually numbers? Maybe we should return them as strings

Comment (hebime) [reply slate/source/includes/rest/core/_app_user.md]: they are numbers in the schema :(

Comment (alavers) [reply slate/source/includes/rest/core/_app_user.md]: Number ids ticking a time bomb, but even still I think we can just pass them through as numbers for now. It's easier on current API consumers who wouldn't have to parse ints.

If hipchat does make a breaking change their API and change them to strings, everyone will already be broken anyway, not just our API consumers. Any fix on our part would be straightforward.

This is something we probably won't have to worry about. Hipchat as a smooch channel is pretty much dead. Even more so with the Atlassian Stride announcement.

File: slate/source/includes/rest/core/_app_user.md [hunk 3]
```diff
@@ -308,7 +310,52 @@ curl https https://api.smooch.io/v1/apps/5963c0d619a30a2e00de36b8/appusers/deb92
...
+            "type": "helpscout",
+            "conversationId": "123456"
+        }
+    ]
+}
+```
+
+<api>`GET /v1/apps/{appId}/appusers/{smoochId|userId}/businesssystems`</api>
 
+Retrieves all of the app user's business system IDs.
```
Comment (jpjoyal) [inline slate/source/includes/rest/core/_app_user.md]: Let's add a table to document type and the possible id keys

File: slate/source/includes/rest/core/_app_user.md [hunk 4]
```diff
@@ -308,7 +310,52 @@ curl https https://api.smooch.io/v1/apps/5963c0d619a30a2e00de36b8/appusers/deb92
...
```
Comment (jpjoyal) [inline slate/source/includes/rest/core/_app_user.md]: Retrieves all the [business systems](https://app.smooch.io/integrations/categories/business-systems) a appUser's conversation is connected to.

**Pull request merged**